.github/workflows/feature-serverless-framework-deployment.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       - name: Set up Go 1.19
```

Comment:
```diff
-  - name: Set up Go 1.19
+  - name: Set up Go 1.21
```

Reply:
Thanks for spotting this, I've just corrected it